daily-challenges/2024-09/17/lcaloi_29.spec.ts

```diff
@@ -0,0 +1,85 @@
+//Javascript
+import { test, expect, Page } from "@playwright/test";
+
+function isPrime(number: number) {
+    if (number > 2 && number % 2 === 0 || number <= 1) {
```

Comment:
Mặc dù không sai, nhưng hãy thêm ngoặc vào nhé bro.
Viết code làm sao để đồng đội không phải hỏi câu nào, hoặc ko phải confuse.
```suggestion
    if ((number > 2 && number % 2 === 0) || number <= 1) {
```